roslibrust/src/ros1/node/actor.rs

```diff
@@ -229,19 +264,19 @@ pub(crate) struct Node {
     publishers: HashMap<String, Publication>,
     // Record of subscriptions this node has
     subscriptions: HashMap<String, Subscription>,
-    // Record of what services this node is serving
-    services: HashMap<String, ServiceCallback>,
```

Comment:
I know this is for service servers, but since that isn't supported for native ROS1 yet, I repurposed it in part to make the unused warning go away.